src/app/app.component.ts

```diff
@@ -65,7 +68,10 @@ export class AppComponent implements OnInit {
 
   ngOnInit(): void {
     this.configService.getConfigurationFromBackend().subscribe(
-      () => (this.loading = false),
+      () => {
+        this.loading = false;
+        this.editingFromConfigServer = this.configService.configuration.features[UiFeatures.EDITING];
```

Comment:
I guess we should add a separate feature that indicates if the PlanQK Logo should be displayed or not.

Reply:
Added a new configuration entry that defaults to true (if the config server is not available) and indicates if planqk-ui should be used